.github/workflows/tidy.yml

```diff
@@ -30,23 +30,22 @@ jobs:
     - uses: actions/checkout@v4
     - name: modules
       run: |
-        sudo apt-get install -y software-properties-common
-        sudo add-apt-repository -y ppa:ginggs/deal.ii-9.4.0-backports
+        # sudo apt-get install -y software-properties-common
+        # sudo add-apt-repository -y ppa:ginggs/deal.ii-9.6.0-backports
```

Comment:
Is there any point in keeping these lines?

Reply:
I left them in case we want to use backports in the future (a package for the 9.6 release does not exist yet).

We can get rid of them now if you like. I don't have a preference.

Reply:
Let's just remove them. It seems we are fine with the current approach.

Reply:
I removed the backports comments.